docs/documentation/make-first-prototype/let-user-change-answers.md

```diff
@@ -0,0 +1,43 @@
+# Let the user change their answers
+
+## Make the ‘Change’ links work
+
+Make the **Change** links on the ‘Confirmation’ page work by adding the right links.
+
+1. In the the <a> tag under `{{ data['how-many-balls'] }}`, change the href attribute from `#` to ‘/juggling-balls’
+2. In the the <a> tag under `{{ data['most-impressive-trick'] }}`, change the href attribute from `#` to ‘/juggling-trick’
```

Comment:
Is `/juggling-trick` meant to be wrapped in backticks rather than quotes?

```suggestion
2. In the the <a> tag under `{{ data['most-impressive-trick'] }}`, change the href attribute from `#` to `/juggling-trick`
```